repos/system_upgrade/el7toel8/actors/kernel/checkkerneldrivers/actor.py

```diff
@@ -52,29 +51,17 @@ def process(self):
                     if active_module.filename in removed_drivers:
                         collected_drivers.add(active_module.filename)
 
-            # Going over the collected drivers and considering for reporting only
-            # those drivers that are currently used by some device.
-            udevadm_db = ''
-            for fact in self.consume(UdevAdmInfoData):
-                udevadm_db += fact.db
-            for line in udevadm_db.split('\n'):
-                if 'E: DRIVER=' in line:
-                    _, driver = line.split('=')
-                    if driver in collected_drivers:
-                        drivers_to_report.add(driver)
-
             # In the end, we are only going to report drivers that are:
             # - removed in the RHEL8 (are part of files/removed_drivers.txt)
             # - not whitelisted
-            # - currently being used by some device
-            if drivers_to_report:
```

Comment:
While this change (removing vetting against actually active kernel drivers) will make new reporting testing a lot easier, I'm wondering if this can theoretically backfire? Like the upgrade will be falsely inhibited although the loaded kernel modules aren't actually used?